__tests__/app.test.js

```diff
@@ -255,105 +327,117 @@ describe('CORE: GET /api/articles' , () => {
-                })
-                .then(() => {
-                    return db.query('DELETE FROM topics;')
-                })
-                .then(() => {
-                    return request(app)
-                    .get('/api/topics')
-                    .expect(404)
-                })
-                .then((response) => {
-                    const error = response.body
-                    expect(error.msg).toBe('No Topics Found')
-                })
+            comments.forEach((comment) => {
+            expect(comment).toMatchObject(exampleCommentObj)
             })
         })
+    })
+    test('the comments array is server in order of date created descending', () => {
+        return request(app)
+        .get('/api/articles/1/comments')
+        .expect(200)
+        .then((response) => {
+            const comments = response.body.comments;
+            expect(comments.length).not.toBe(0);
 
-        describe('Get articlebyID', () => {
-            test('returns a 404 error when the database has no entries for a valid id format but no id in the database', () => {
-                return request(app)
-                .get('/api/articles/100')
-                .expect(404)
-            })
-            test('returns a custom error object of {status: 404, msg: "No Article Found"} if no entries in database', () => {
-                return request(app)
-                .get('/api/articles/100')
-                .expect(404)
-                .then((response) => {
-                    const error = response.body;
-                    expect(error.msg).toBe('No Article Found')
-                })
-            })
+            const orderedComments = [...comments];
+
+            orderedComments.sort((currCom, nextCom) => Date.parse(nextCom.created_at) - Date.parse(currCom.created_at))
+            expect(comments).toEqual(orderedComments);
         })
-        
-        describe('None existant endpoints', () => {
-            test('Returns a 404 status for an endpoint that does not exist', () => {
-                return request(app)
-                .get('/api/does-not-exist')
-                .expect(404)
-            })
-            test('returns a customer error object of {status: 404, msg:"Endpoint Does Not Exist"}', () => {
-                return request(app)
-                .get('/api/does-not-exist')
-                .expect(404)
-                .then((response) => {
-                    const error = response.body;
-                    expect(error.msg).toBe('Endpoint Does Not Exist');
-                })
+    })
+    test('returns a status of 404 and a custom error object of {status:404, msg:"No Comments Found"} if the database has no comments for a valid article_id', () => {
+        return request(app)
+        .get('/api/articles/2/comments')
+        .expect(404)
+        .then((response) => {
+            const error = response.body;
+            expect(error.msg).toBe('No Comments Found');
+        })
```

Comment:
This should be a 200 - if the id is valid, we just want to get a 200 back and an empty array